ci/ray_ci/serve.tests.env.sh

```diff
@@ -3,4 +3,21 @@
 
 set -exo pipefail
 
-true
+# Unset dind settings; we are using the host's docker daemon.
+unset DOCKER_TLS_CERTDIR
+unset DOCKER_HOST
+unset DOCKER_TLS_VERIFY
+unset DOCKER_CERT_PATH
```

Comment:
you probably do not need to unset these. on civ1 pipelines, it is still using dind technically speaking... and because you are running with your concurrent docker wrapper thing, these env vars are not set here anyways.